src/components/molecules/LocationMap.tsx

```diff
@@ -1,32 +1,72 @@
 import 'leaflet-css'
 import React, { FunctionComponent } from 'react'
 import L from 'leaflet'
-import { Map, Marker, TileLayer } from 'react-leaflet'
+import { Map, TileLayer } from 'react-leaflet'
+import AnywayMarker from '../atoms/AnywayMarker'
 
 // TODO: Move to init.service once it's merged.
 L.Icon.Default.mergeOptions({
     iconRetinaUrl: require('leaflet/dist/images/marker-icon-2x.png'),
     iconUrl: require('leaflet/dist/images/marker-icon.png'),
     shadowUrl: require('leaflet/dist/images/marker-shadow.png')
 })
+const lIconSizes = { iconSize: [25, 41], iconAnchor: [12, 41], popupAnchor: [1, -34], shadowSize: [41, 41] }
+const iconSize = lIconSizes;
+
 
 interface IProps {
+    data: any[],
     marker: { lat: number; lng: number }
 }
 
 const INITIAL_ZOOM = 13
 const WRAPPER_STYLES = { height: '100%', width: '100%' }
 
-const LocationMap: FunctionComponent<IProps> = ({ marker }) => {
+const LocationMap: FunctionComponent<IProps> = ({ data, marker }) => {
+    let markers = data.map((x: any) => {
+        if (x.latitude !== null && x.longitude !== null) {
+            return <AnywayMarker markerdata={x} />
+        }
+        else return null;
```

Comment:
it i move the else line i get a warning

Reply:
Really? Which warning?

Reply:
Oh? Which warning?

Reply:
Expected to return a value at the end of arrow function

Reply:
True, but you don't have to write `else` before you return it.
Simply a `return null` works here, because when you return in the `if`, there's no way that the two blocks of code will run :)